Control/Monad/Logic/Class.hs

```diff
@@ -165,26 +181,110 @@ class (Monad m, Alternative m) => MonadLogic m where
     (>>-)      :: m a -> (a -> m b) -> m b
     infixl 1 >>-
 
-    -- | Logical __conditional.__ The equivalent of
-    --   <http://lpn.swi-prolog.org/lpnpage.php?pagetype=html&pageid=lpn-htmlse44 Prolog's soft-cut>.
-    --   If its first argument succeeds at all, then the results will be fed into
-    --   the success branch. Otherwise, the failure branch is taken.
-    --   satisfies the following laws:
-    --
-    --   > ifte (pure a) th el       == th a
-    --   > ifte empty th el          == el
-    --   > ifte (pure a <|> m) th el == th a <|> (m >>= th)
-    ifte       :: m a -> (a -> m b) -> m b -> m b
-
     -- | __Pruning.__ Selects one result out of many. Useful for when multiple
     --   results of a computation will be equivalent, or should be treated as
     --   such.
+    --
+    --   As an example, here's a way to determine if a number is
+    --   composite (has non-trivial integer division, i.e. the not a
```

Comment:
```suggestion
    --   composite (has non-trivial integer divisor, i.e. not a
```

Reply:
Ugh, that's what I get for trying to dual task and edit sentences while in a video conference.  Thanks for catching  that.